src/main/java/perkpack/models/Card.java

```diff
@@ -15,7 +15,7 @@
     @JoinTable
     private Set<Account> accounts = new HashSet<>();
 
-    @OneToMany(mappedBy ="cardPerkBelongsTo", cascade = CascadeType.ALL, fetch = FetchType.EAGER)
```

Comment:
What was wrong with this

Reply:
The perk's card wasn't being set with that annotation. I just copied what we had for categories.